index.js

```diff
@@ -169,9 +165,26 @@ module.exports = function(source, map) {
 			css.code += '\n/*# sourceMappingURL=' + css.map.toUrl() + '*/';
 			js.code = js.code + `\nimport '${posixify(cssFilepath)}';\n`;
 
-			if (virtualModules) {
-				virtualModules.writeModule(cssFilepath, css.code);
+			if (!virtualModules) {
+				throw new Error(
+					`
+
+					To be able to use emitCss: true, add
+
+						const SveltePlugin = require('svelte-loader').plugin;
+
+					to the top of your webpack.config.js and
+					SveltePlugin to the plugins array like this
+
+						plugins: [
+							...
+							new SveltePlugin()
+						]
+					`.split('\n').map(s => s.slice(5)).join('\n')
```

Comment:
It would be helpful to indicate that if you're running in watch mode a server restart is needed

```suggestion

					If you are running Webpack in watch mode, restart the server to apply these changes.
					`.split('\n').map(s => s.slice(5)).join('\n')
```